Invoro.WebApp/Invoro/invoro/src/lib/components/Vote/Vote.component.test.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { mount, ReactWrapper } from 'enzyme';
+import VoteComponent from './Vote.component';
+
+describe("VoteComponent", () => {
+    let voteComponent: ReactWrapper;
+
+    describe("Star Icon rendered when isVoted is true", () => {
+        beforeEach(() => {
+            let props = {
+                isVoted: true,
+                voteHandle: () => { },
+                unvoteHandle: () => { },
+            };
+            voteComponent = mount(<VoteComponent {...props} />);
+        });
+
+        it("isVoted true Exactly one icon is rendered", () => {
```

Comment:
This test case it not related to Vote component. It is related to VoteIcon Component.
As for Vote.component, it doesn't need to know that the 'StartIcon' is about to be renderd.
Remove this test from this class or move it to VoteIcon.component tests class.

Reply:
I tested VoteComponent. VoteComponent renders an IconButton. 
I expect anyway to render exactly one IconButton (which kind of IconButton depends on the value of isVoted). This test just checks that exactly one IconButton is rendered.